### PR TITLE
left rotation.cpp

### PR DESCRIPTION
**Pull request by aatif547**

### CODE REVIEW

**Review by VenkatVuddagiri — COMMENTED** [review]
Add comments to your code for more readability

File: Write_Program_in_any_lang/C++/Left rotation [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#include<iostream>
+using namespace std;
+int main()
+{
+    int n,d;
+    cin>>n;
```
Comment (aatif547) [inline Write_Program_in_any_lang/C++/Left rotation:6]: taking a user input

**Review by aatif547 — COMMENTED** [review]
comment added

**Review by VenkatVuddagiri — APPROVED** [review]
Awesome

**Comment by aatif547** [comment]
kindly accept my pull request

**Comment by VenkatVuddagiri** [comment]
It will be merged buddy, don't worry.

**Pull request merged**